resources/assets/coffee/react/beatmap-discussions/nominations.coffee

```diff
@@ -33,19 +33,45 @@ class BeatmapDiscussions.Nominations extends React.PureComponent
   componentDidUpdate: =>
     osu.pageChange()
 
+  focusHypeInput: ->
+    hypeMessage = $('.beatmap-discussion-new__footer-content')[0]
+    inputBox = $('.beatmap-discussion-new__message-area')
+    flashClass = 'beatmap-discussion-new__footer-content--flash'
+
+    # switch to generalAll tab, set current filter to praises
+    $.publish 'beatmapDiscussion:setMode', mode: 'generalAll'
+    $.publish 'beatmapDiscussion:filter', filter: 'praises'
+
+    # ensure input box is in view and focus it
+    $.scrollTo(inputBox[0], 200)
```

Comment:
need some offset otherwise it'll be covered by page switcher.

![](https://s.myconan.net/2017-10/chrome_2017-10-18_17-05-41.png)

And remember this stupid thing (maybe just hide the fixed bar on mobile) :D 

![](https://s.myconan.net/2017-10/chrome_2017-10-18_17-07-57.png)